File: app/src/main/java/com/toperc/aatools/MainActivity.java
package com.toperc.aatools;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.toperc.lib_aatools.AAToolsMgr;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.open_btn).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                AAToolsMgr.open(MainActivity.this);
            }
        });

        findViewById(R.id.test_crash_btn).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                int a = 1 / 0;
            }
        });

    }
}

File: app/src/main/java/com/ixinrun/aatools/MainActivity.kt
package com.ixinrun.aatools

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.ixinrun.lib_aatools.AATools

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<View>(R.id.open_btn).setOnClickListener(object : View.OnClickListener {
                override fun onClick(v: View?) {
                    AATools.open(this@MainActivity)
                }
            })

        findViewById<View>(R.id.test_crash_btn).setOnClickListener(object : View.OnClickListener {
                override fun onClick(v: View?) {
                    val a = 1 / 0
                }
            })
    }
}